packages/storage/tsconfig.json

```diff
@@ -4,6 +4,7 @@
     "outDir": "dist"
   },
   "exclude": [
-    "dist/**/*"
+    "dist/**/*",
+    "karma.conf.js"
```

Comment:
Is it needed? TS should ignore it by default since it's JS.

Reply:
This was part of an attempt to try to get eslint to stop complaining about js files not following Typescript rules, but it doesn't work anyway so removed.